lib/active_merchant/billing/gateways/ogone.rb

```diff
@@ -334,6 +334,7 @@ def add_address(post, creditcard, options)
       def add_invoice(post, options)
         add_pair post, 'orderID', options[:order_id] || generate_unique_id[0...30]
         add_pair post, 'COM',     options[:description]
+        add_pair post, 'ORIG', options[:origin] if options[:origin]
```

Comment:
Tiny nitpick, align `options` to match indentation context (or unalign everything else 😅 )